crates/bevy_pbr/src/material_bind_groups.rs

```diff
@@ -699,7 +702,10 @@ where
                     bindless_buffer_descriptor.bindless_index,
                     render_device.create_buffer(&BufferDescriptor {
                         label: Some("bindless fallback buffer"),
-                        size: bindless_buffer_descriptor.size as u64,
+                        size: match bindless_buffer_descriptor.size {
+                            Some(size) => size as u64,
+                            None => DEFAULT_BINDLESS_FALLBACK_BUFFER_SIZE,
+                        },
```

Comment:
What is the consequence of this?

Reply:
We need some size for the fallback buffer, and we can't infer a size with material storage buffers because they're just untyped `Buffer`s (not `BufferVec<T>` or anything) in the material structure. So I just picked an arbitrary small size. I tested that it works even if the actual structures stored inside the buffer have different sizes.

Reply:
I was considering allowing the developer to specify an explicit fallback size with `#[storage(10, fallback_size(32))]` or whatever, but I decided against supporting that because I didn't see any cases in which it'd be needed.

Reply:
Might be worth just adding a comment that this was chosen with some testing but is arbitrary to the constant definition.

Reply:
Done.